src/project/internal/projectuiactions.cpp

```diff
@@ -38,23 +38,23 @@ const UiActionList ProjectUiActions::m_actions = {
              IconCode::Code::NEW_FILE
              ),
     UiAction("file-close",
-             mu::context::UiCtxNotationOpened,
+             mu::context::UiCtxNotationOrPublishOpened,
              QT_TRANSLATE_NOOP("action", "Close"),
              QT_TRANSLATE_NOOP("action", "Close current score")
              ),
     UiAction("file-save",
-             mu::context::UiCtxNotationOpened,
+             mu::context::UiCtxNotationOrPublishOpened,
```

Comment:
Looks like now the user won't be able to save their score from the home page

Reply:
That was already the case, and will indeed also be the case after this PR. Personally, I could see a way that this makes sense from a UX point of view, because on the home page, you don't _see_ the score, and on the other pages you do. Not a very strong point though, so if you think these actions should also be available on the homepage too, then I'll change them to `UiCtxAny`.

Reply:
>Not a very strong point though, so if you think these actions should also be available on the homepage too, then I'll change them to UiCtxAny.

@Tantacrul what do you think?

Reply:
UiCtxAny is not suitable in this case, since these actions should be blocked if the user does not have any score open

Reply:
I would recommend removing all these changes from your PR and keeping only what is relevant to the original issue

Reply:
Yeah, that's the best for now. Done!